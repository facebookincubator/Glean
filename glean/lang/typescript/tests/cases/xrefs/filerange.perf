{
  "@generated": null,
  "facts_searched": { "scip.FileRange.1": 62 },
  "full_scans": [ "scip.FileRange.1" ]
}