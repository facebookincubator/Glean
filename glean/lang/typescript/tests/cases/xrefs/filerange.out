[
  "@generated",
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 1,
        "columnBegin": 1,
        "lineEnd": 1,
        "columnEnd": 1
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 9,
        "columnBegin": 1,
        "lineEnd": 53,
        "columnEnd": 1
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 9,
        "columnBegin": 8,
        "lineEnd": 9,
        "columnEnd": 9
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 10,
        "columnBegin": 8,
        "lineEnd": 10,
        "columnEnd": 9
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 11,
        "columnBegin": 8,
        "lineEnd": 11,
        "columnEnd": 11
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 12,
        "columnBegin": 8,
        "lineEnd": 12,
        "columnEnd": 12
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 14,
        "columnBegin": 1,
        "lineEnd": 42,
        "columnEnd": 3
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 14,
        "columnBegin": 7,
        "lineEnd": 14,
        "columnEnd": 9
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 15,
        "columnBegin": 3,
        "lineEnd": 15,
        "columnEnd": 13
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 15,
        "columnBegin": 3,
        "lineEnd": 33,
        "columnEnd": 3
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 15,
        "columnBegin": 23,
        "lineEnd": 15,
        "columnEnd": 25
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 16,
        "columnBegin": 11,
        "lineEnd": 16,
        "columnEnd": 13
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 16,
        "columnBegin": 17,
        "lineEnd": 16,
        "columnEnd": 21
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 16,
        "columnBegin": 41,
        "lineEnd": 16,
        "columnEnd": 43
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 16,
        "columnBegin": 46,
        "lineEnd": 16,
        "columnEnd": 48
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 16,
        "columnBegin": 51,
        "lineEnd": 16,
        "columnEnd": 56
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 17,
        "columnBegin": 9,
        "lineEnd": 17,
        "columnEnd": 11
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 18,
        "columnBegin": 17,
        "lineEnd": 18,
        "columnEnd": 21
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 18,
        "columnBegin": 52,
        "lineEnd": 18,
        "columnEnd": 54
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 19,
        "columnBegin": 11,
        "lineEnd": 19,
        "columnEnd": 13
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 20,
        "columnBegin": 11,
        "lineEnd": 20,
        "columnEnd": 13
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 23,
        "columnBegin": 5,
        "lineEnd": 23,
        "columnEnd": 9
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 24,
        "columnBegin": 7,
        "lineEnd": 24,
        "columnEnd": 9
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 24,
        "columnBegin": 12,
        "lineEnd": 24,
        "columnEnd": 14
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 25,
        "columnBegin": 7,
        "lineEnd": 25,
        "columnEnd": 12
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 27,
        "columnBegin": 5,
        "lineEnd": 27,
        "columnEnd": 9
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 27,
        "columnBegin": 48,
        "lineEnd": 27,
        "columnEnd": 50
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 27,
        "columnBegin": 53,
        "lineEnd": 27,
        "columnEnd": 55
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 27,
        "columnBegin": 58,
        "lineEnd": 27,
        "columnEnd": 63
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 29,
        "columnBegin": 5,
        "lineEnd": 29,
        "columnEnd": 9
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 30,
        "columnBegin": 7,
        "lineEnd": 30,
        "columnEnd": 9
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 30,
        "columnBegin": 12,
        "lineEnd": 30,
        "columnEnd": 14
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 31,
        "columnBegin": 7,
        "lineEnd": 31,
        "columnEnd": 12
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 34,
        "columnBegin": 3,
        "lineEnd": 34,
        "columnEnd": 8
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 34,
        "columnBegin": 3,
        "lineEnd": 41,
        "columnEnd": 5
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 34,
        "columnBegin": 10,
        "lineEnd": 34,
        "columnEnd": 12
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 34,
        "columnBegin": 23,
        "lineEnd": 34,
        "columnEnd": 26
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 34,
        "columnBegin": 37,
        "lineEnd": 34,
        "columnEnd": 42
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 35,
        "columnBegin": 11,
        "lineEnd": 35,
        "columnEnd": 16
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 35,
        "columnBegin": 20,
        "lineEnd": 35,
        "columnEnd": 24
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 35,
        "columnBegin": 45,
        "lineEnd": 35,
        "columnEnd": 47
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 35,
        "columnBegin": 55,
        "lineEnd": 35,
        "columnEnd": 57
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 35,
        "columnBegin": 60,
        "lineEnd": 35,
        "columnEnd": 65
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 36,
        "columnBegin": 9,
        "lineEnd": 36,
        "columnEnd": 14
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 37,
        "columnBegin": 17,
        "lineEnd": 37,
        "columnEnd": 21
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 37,
        "columnBegin": 51,
        "lineEnd": 37,
        "columnEnd": 56
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 38,
        "columnBegin": 11,
        "lineEnd": 38,
        "columnEnd": 16
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 39,
        "columnBegin": 11,
        "lineEnd": 39,
        "columnEnd": 16
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 46,
        "columnBegin": 1,
        "lineEnd": 52,
        "columnEnd": 1
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 46,
        "columnBegin": 17,
        "lineEnd": 46,
        "columnEnd": 30
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 46,
        "columnBegin": 36,
        "lineEnd": 46,
        "columnEnd": 42
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 46,
        "columnBegin": 53,
        "lineEnd": 46,
        "columnEnd": 55
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 46,
        "columnBegin": 58,
        "lineEnd": 46,
        "columnEnd": 60
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 47,
        "columnBegin": 9,
        "lineEnd": 47,
        "columnEnd": 15
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 47,
        "columnBegin": 19,
        "lineEnd": 47,
        "columnEnd": 20
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 47,
        "columnBegin": 34,
        "lineEnd": 47,
        "columnEnd": 37
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 47,
        "columnBegin": 44,
        "lineEnd": 47,
        "columnEnd": 45
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 49,
        "columnBegin": 9,
        "lineEnd": 49,
        "columnEnd": 11
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 49,
        "columnBegin": 19,
        "lineEnd": 49,
        "columnEnd": 21
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 49,
        "columnBegin": 23,
        "lineEnd": 49,
        "columnEnd": 29
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 51,
        "columnBegin": 11,
        "lineEnd": 51,
        "columnEnd": 17
      }
    }
  },
  {
    "key": {
      "file": { "key": "example.ts" },
      "range": {
        "lineBegin": 51,
        "columnBegin": 20,
        "lineEnd": 51,
        "columnEnd": 22
      }
    }
  }
]